indra/newview/llimview.cpp

```diff
@@ -3241,7 +3248,7 @@ void LLIMMgr::addMessage(
             }
 
             //Play sound for new conversations
-            if (!skip_message & !gAgent.isDoNotDisturb() && (gSavedSettings.getBOOL("PlaySoundNewConversation")))
+            if (!skip_message && !gAgent.isDoNotDisturb() && (gSavedSettings.getBOOL("PlaySoundNewConversation")))
```

Comment:
IMs aren't particularly frequent, but probably better to use LLCachedControl here.